spring-boot-examples/workflows/src/main/java/io/dapr/springboot/examples/wfp/fanoutin/FanOutInWorkflow.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2023 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
same for this filename: `fanoutin` -> `faninout`

Reply:
@cicoyle the example is fanning out tasks to then do a fan-in.. I think the original example name was wrong.

Reply:
This is also aligned with how the Dapr University is doing it: https://github.com/dapr/quickstarts/tree/master/tutorials/workflow/csharp/fan-out-fan-in

Reply:
[since our docs have out first](https://docs.dapr.io/developing-applications/building-blocks/workflow/workflow-patterns/#fan-outfan-in), Im fine keeping it as is then. There appears to be some inconsistency with the naming convention.